bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/config/CredentialFactory.java

```diff
@@ -22,6 +22,7 @@
 import com.google.auth.Credentials;
```

Comment:
Credentials is no longer needed. Why do we need OAuth2Credentials instead of Credentials?